src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/table.svelte

```diff
@@ -96,6 +165,26 @@
         {#each data.documents.documents as document}
             <TableRowLink
                 href={`${base}/console/project-${projectId}/databases/database-${databaseId}/collection-${$collection.$id}/document-${document.$id}`}>
+                <TableCell>
+                    <InputCheckbox
+                        id="select-{document.$id}"
+                        value={selectedDb.includes(document.$id)}
+                        on:click={(e) => {
+                            // Prevent the link from being followed
+                            e.preventDefault();
+                            const el = e.currentTarget;
+                            if (!isHTMLInputElement(el)) return;
+
+                            selectedDb = toggle(selectedDb, document.$id);
+
+                            // Hack to make sure the checkbox is checked, independent of the
+                            // preventDefault() call above
+                            window.setTimeout(() => {
+                                el.checked = selectedDb.includes(document.$id);
+                            });
+                        }} />
+                </TableCell>
```

Comment:
@TGlide same comment about the cellHead, if we use this often we should probably create a component for it to handle all the logic